src/redux/reducers/samples/samplesCreate.js

```diff
@@ -2,7 +2,7 @@ const samplesCreate = (state, action) => {
   const { sample } = action.payload;
   return {
     ...state,
-    ids: [...state.ids, sample.uuid],
+    // ids: [...state.ids, sample.uuid],
```

Comment:
If we're deleting samples, better just delete them instead of commenting them out?